run.sh

```diff
@@ -346,6 +349,25 @@ check_plugins() {
   done <<< "$(WP plugin list --field=name)"
 }
 
+check_packages() {
+  # If a composer.json file exists in /app => install it
+  if [[ -f "/app/composer.json" ]]; then
+    h3 "Installing packages listed in composer.json"
+    composer install
+    return
+  fi
+
+  if [[ "${REQUIRE-}" ]]; then
```

Comment:
Yes but let's say you have a `composer.json` file and left a REQUIRE by mistake. It will overwrite your `composer.json` file. I don't think that's a behaviour we want.